src/components/AgreementLineSections/Coverage/Coverage.test.js

```diff
@@ -0,0 +1,157 @@
+import Coverage from './Coverage';
+
+describe('Coverage', () => {
+    describe('renders with coverage type custom ', () => {
+      beforeEach(() => {
+        renderWithIntl(
+          <Coverage line={line} resource={resource} />,
+          translationsProperties
+        );
+      });
+
+      test('renders Coverage Accordion', async () => {
+        await Accordion('Coverage').exists();
+      });
+
+      test('renders Coverage Accordion', async () => {
+        await KeyValue('Embargo').exists();
+      });
+
+      test('renders the expected embargo value', async () => {
+        await KeyValue('Embargo').has({ value: 'Moving wall end: 4 years' });
+      });
+
+      test('renders expected column count', async () => {
+        await MultiColumnList({ columnCount: 7 }).exists();
+      });
+
+      test('renders expected columns', async () => {
+        await MultiColumnList({
+          columns: [
+            'Start date',
+            'Start volume',
+            'Start issue',
+            'End date',
+            'End volume',
+            'End issue',
+            'Coverage type',
+          ],
+        }).exists();
+      });
+
+      test('renders expected coverage start date in the row', async () => {
+        Promise.all([
+          await MultiColumnListCell({ row: 0, columnIndex: 0 }).has({ content: '9/1/2021' }),
+          await MultiColumnListCell({ row: 1, columnIndex: 0 }).has({ content: '10/1/2021' })
+        ]);
+      });
+
+      test('renders expected coverage start volume in the row', async () => {
+        Promise.all([
+          await MultiColumnListCell({ row: 0, columnIndex: 1 }).has({ content: '1' }),
+          await MultiColumnListCell({ row: 1, columnIndex: 1 }).has({ content: '1' })
+        ]);
+      });
+
+      test('renders expected coverage start issue in the row', async () => {
+        Promise.all([
+          await MultiColumnListCell({ row: 0, columnIndex: 2 }).has({ content: '1' }),
+          await MultiColumnListCell({ row: 1, columnIndex: 2 }).has({ content: '1' })
+        ]);
+      });
+
+      test('renders expected coverage end date in the row', async () => {
+        Promise.all([
+          await MultiColumnListCell({ row: 0, columnIndex: 3 }).has({ content: '9/30/2021' }),
+          await MultiColumnListCell({ row: 1, columnIndex: 3 }).has({ content: '10/30/2021' })
+        ]);
+      });
+
+      test('renders expected coverage end volume in the row', async () => {
+        Promise.all([
+          await MultiColumnListCell({ row: 0, columnIndex: 4 }).has({ content: '8' }),
+          await MultiColumnListCell({ row: 1, columnIndex: 4 }).has({ content: '8' })
+        ]);
+      });
+
+      test('renders expected coverage end issue in the row', async () => {
+        Promise.all([
+          await MultiColumnListCell({ row: 0, columnIndex: 5 }).has({ content: '12' }),
+          await MultiColumnListCell({ row: 1, columnIndex: 5 }).has({ content: '12' })
```

Comment:
Can we have different content values for individual rows, because having the same content value for both the rows could cause false positives

Reply:
sure

Reply:
I changed the values.